internal/bminventory/inventory.go

```diff
@@ -2186,6 +2198,16 @@ func (b *bareMetalInventory) refreshHostStatus(
 	return nil
 }
 
+func (b *bareMetalInventory) setMajorityGroupForCluster(clusterID *strfmt.UUID, db *gorm.DB) error {
+	var cluster common.Cluster
+	var err error
+
+	if err = db.Select("id, status").Take(&cluster, "id = ?", clusterID.String()).Error; err != nil {
```

Comment:
Maybe the get can be done inside `SetConnectivityMajorityGroupsForCluster`?

Reply:
OK